cypress/obj/LoginPage.js

```diff
@@ -0,0 +1,26 @@
+class LoginPageVariables {
+
+    getRootPage(){
+        cy.visit('/')
```

Comment:
from function naming conventions if you start function name with "get..." you would expect it to return something. This one is more a "navigate" type of functions